modules/developer-cli-odo-installing-odo-on-linux.adoc

```diff
@@ -7,27 +7,28 @@
 
-The `{odo-title}` CLI is available to download as a binary and as a tarball for multiple operating systems and architectures including: 
+The `{odo-title}` CLI is available to download as a binary and as a tar file, for multiple operating systems and architectures including: 
 
-[cols="2,1,1",options="header"]
+[cols="1,1,1",options="header"]
 |===
-|Operating System|Binary|Tarball
-|Linux|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-amd64[odo-linux-amd64] |link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-amd64.tar.gz[odo-linux-amd64.tar.gz]
-|Linux on IBM Power|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-ppc64le[odo-linux-ppc64le] |link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-ppc64le.tar.gz[odo-linux-ppc64le.tar.gz]
-|Linux on IBM Z and LinuxONE|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-s390x[odo-linux-s390x] |link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-s390x.tar.gz[odo-linux-s390x.tar.gz]
+|Operating system|Binary|Tar file
+|Linux on AMD|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-amd64[`odo-linux-amd64`] |link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-amd64.tar.gz[`odo-linux-amd64.tar.gz`]
+|Linux on ARM|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-arm64[`odo-linux-arm64`] |link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-arm64.tar.gz[`odo-linux-arm64.tar.gz`]
+|Linux on IBM Power|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-ppc64le[`odo-linux-ppc64le`] |link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-ppc64le.tar.gz[`odo-linux-ppc64le.tar.gz`]
+|Linux on IBM Z and LinuxONE|link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-s390x[`odo-linux-s390x`] |link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/odo-linux-s390x.tar.gz[`odo-linux-s390x.tar.gz`]
 |===
 
 
 .Procedure
 
-. Navigate to the link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/[content gateway] and download the appropriate file for your operating system and architecture.
+. Browse to the link:https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/latest/[content gateway] and download the appropriate file for your operating system and architecture.
```

Comment:
See the IBM SG "Referring to web addresses" section:

Use the following verbs and prepositions with web addresses:
To direct readers to a website or web page, use see or go to.
For more information about the product, see CICS Explorer
.
To obtain videos for software support, go to the IBM Software Subscription and Support
website.
To describe where information is located, use on.
See the latest product information on the CICS Explorer
website.
Find information about database management systems on the IBM Db2 Database
page.
To describe where to download a file, use from.
Download the video from the IBM Software Subscription and Support
website.

Reply:
Ok, I will change to "Go to the content gateway and download...." seeing as that is what the IBM SG says

Reply:
Done